resources/views/components/button/index.blade.php

```diff
@@ -0,0 +1,11 @@
+{{--
+This is the default button if you use the example you get this.
+Example:
+
+```
+<x-rapidez::button>Something</x-rapidez::button>
+```
+--}}
+<x-rapidez::button.base {{ $attributes->twMerge('relative inline-flex items-center justify-center text-white transition bg text-base font-medium rounded min-h-12 py-1.5 px-5 hover:bg-opacity-80 border-b border-black/15 disabled:text-muted disabled:bg-disabled disabled:cursor-not-allowed') }}>
```

Comment:
- `disabled:bg-disabled` should not exists anymore => `bg-muted`?
- `hover:bg-opacity-80` should we change this to `bg-emphasis`?
- `border-black/15`, do we want/need to be explicit and not use the border colors? Just `border`?

So this becomes: `border bg disabled:bg-muted hover:bg-emphasis`?